### PR TITLE
Czech language support fixed and improved

### PR DESCRIPTION
**Pull request by foton**
Continued on work from https://github.com/kslazarev/numbers_and_words/pull/123.

### CODE REVIEW

File: spec/numbers_and_words/array/fixture_examples/cs.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+to_words:
+  # simple_example:
```
Comment (jlduran) [inline spec/numbers_and_words/array/fixture_examples/cs.yml]: Please uncomment/remove these specs.

File: spec/numbers_and_words/array/fixture_examples/cs.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - 1001
+      - 101
+      - 21
+      - 13
+    :
+      - jeden tisíc jedna
+      - jedno sto jedna
+      - dvacet jedna
+      - třináct
+  # complex_example_with_options:
```
Comment (jlduran) [inline spec/numbers_and_words/array/fixture_examples/cs.yml]: Please uncomment/remove these specs.

File: spec/numbers_and_words/array/fixture_examples/cs.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  #   options:
+  #     :remove_zero: true
+  #   ?
+  #     - 0.1
+  #     - 0.31
+  #     - 0.12
+  #   :
+  #     - nula celá jedna desetina
```
Comment (jlduran) [inline spec/numbers_and_words/array/fixture_examples/cs.yml]: I guess, with `options: :remove_zero`, this should be just _jedna desetina_.

File: lib/numbers_and_words/strategies/figures_converter/languages/cs.rb [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+          def megs(*args)
+            super({ number: @figures.number_in_capacity(@current_capacity) }.merge(args.first || {}))
+          end
+
+          def gender
+            @current_capacity ||= 0
+            # @current_capacity = order of block of 3 digits, backwards
+
+            case @current_capacity
```
Comment (jlduran) [inline lib/numbers_and_words/strategies/figures_converter/languages/cs.rb]: I guess this `case` could be replaced with a multiple `if` statement, for instance:

```ruby
if @current_capacity.zero?
  options.gender.result
elsif @current_capacity > 2 && @current_capacity.odd?
  :female
else
  :male
end
```

File: spec/numbers_and_words/float/fixture_examples/cs.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+to_words:
+  fractional:
+    0.1: nula celých jedna desetina
+    0.7: nula celých sedm desetin
+    0.01: nula celých jedna setina
+    0.77: nula celých sedmdesát sedm setin
+    0.777: nula celých sedm set sedmdesát sedm tisícin
+    0.7777: nula celých sedm tisíc sedm set sedmdesát sedm desetitisícin
+    0.293517431: nula celých dvě stě devadesát tři milióny pět set sedmnáct tisíc čtyři sta třicet jedna miliardtina
```
Comment (jlduran) [inline spec/numbers_and_words/float/fixture_examples/cs.yml]: In order to get 100% coverage, we need a test like this, for example:

```yml
0.21021: nula celých dvacet jeden tisíc dvacet jedna stotisícina
```

File: spec/numbers_and_words/integer/fixture_examples/cs.yml [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+      :ordinal: true
+      :gender: female
+    2: druhá
+    3: třetí
+    8: osmá
+    18: osmnáctá
+    36: třicátá šestá
+    100: stá
+    999: devítistá devadesátá devátá
+
```
Comment (jlduran) [inline spec/numbers_and_words/integer/fixture_examples/cs.yml]: Please remove this extra line.

File: lib/numbers_and_words/translations/cs.rb [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+      def tens_with_ones(numbers_arr, options = {})
+        connector = ' '
+        [tens(numbers_arr[1], options), connector, ones(numbers_arr[0], options)].join
+      end
+
+      def hundreds(number, options = {})
+        t([options[:prefix], :hundreds, gender_to_use(options)].join('.'))[number]
+      end
+
+      def thousands(number, options = {})
```
Comment (jlduran) [inline lib/numbers_and_words/translations/cs.rb]: In order to achieve 100% coverage, this method should be removed/tested.

**Comment by dblock** [comment]
@foton want to finish this?

**Comment by jlduran** [comment]
@dblock it's OK to merge as is. I can address the suggestions on the weekend.

**Comment by dblock** [comment]
There're a lot of problems in this code, build doesn't pass, etc!

**Comment by dblock** [comment]
I see now, ignore me. I think the change requests should be addressed before merging.

**Comment by foton** [comment]
I will fix it today

**Comment by foton** [comment]
Fixed and coverage is back to 100%.

**Comment by jlduran** [comment]
Thank you @foton!

**Comment by jlduran** [comment]
Closes #123

**Pull request merged**